web-client/src/presenter/computeds/blockedCasesReportHelper.js

```diff
@@ -32,12 +33,28 @@ export const blockedCasesReportHelper = (get, applicationContext) => {
     blockedCasesFormatted = blockedCases
       .sort(applicationContext.getUtilities().compareCasesByDocketNumber)
       .map(blockedCase => {
+        const inConsolidatedGroup = !!blockedCase.leadDocketNumber;
+        const inLeadCase =
+          inConsolidatedGroup &&
+          blockedCase.leadDocketNumber === blockedCase.docketNumber;
+
+        if (inConsolidatedGroup) {
+          if (inLeadCase) {
+            consolidatedIconTooltipText = 'Lead case';
+          } else {
+            consolidatedIconTooltipText = 'Consolidated case';
+          }
+        }
```

Comment:
This seems to be a little more complicated than it needs to be since `inLeadCase` already includes `inConsolidatedGroup` on line 38.
```suggestion
        consolidatedIconTooltipText = inConsolidatedGroup && 'Consolidated case';
        consolidatedIconTooltipText = inLeadCase && 'Lead case';
```

Honestly I still find this syntax a bit esoteric, but it keeps the nested if away. Alternatively, you could leave out line 38 and leave this code as-is or even make a slight change to the code I've proposed and achieve the same thing.